great-idea/services.html

```diff
@@ -1,106 +1,132 @@
+                    </p>
+
+                    <button>Learn More </button>
+                </div>
+
+                <div class='service'>
+                    <h2>Digital Marketing </h2>
+                    <p>
+                        Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                        eleifend. Mauris euismod facilisis iaculis.
+                    </p>
+                    <button>Learn More </button>
+                </div>
+
+                <div class='service'>
+                    <h2>Web Development </h2>
+                    <p>
+                        Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                        eleifend. Mauris euismod facilisis iaculis.
+                    </p>
+
+
+                    <button>Learn More </button>
+                </div>
+
+                <div class='service'>
+                    <h2>iOS Development </h2>
+                    <p>
+                        Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                        eleifend. Mauris euismod facilisis iaculis.
+                    </p>
+
+                    <button>Learn More </button>
+                </div>
+
+                <div class='service'>
+                    <h2>Android Development </h2>
+                    <p>
+                        Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                        eleifend. Mauris euismod facilisis iaculis.
+                    </p>
+
+                    <button>Learn More </button>
+                </div>
+            </div>
+
+
+        </section>
+        <section class="bottom">
+
+            <div>
+                <h3>Some Facts About Our Services </h3>
+                <p>
+                    Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.
+                </p>
+
+                <ul>
+                    <li>Awesome thing</li>
```

Comment:
Just need to align the list a little more to the right, just under the paragraph. You can also change the circles to squares in css.